airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py

```diff
@@ -174,11 +174,7 @@ def account(self) -> AdAccount:
     def _find_account(account_id: str) -> AdAccount:
         """Actual implementation of find account"""
         try:
-            accounts = fb_user.User(fbid="me").get_ad_accounts()
-            for account in accounts:
-                if account["account_id"] == account_id:
-                    return account
+            return AdAccount(f"act_{account_id}")
         except FacebookRequestError as exc:
```

Comment:
Does the exception raised remain a `FacebookRequestError` if the `AdAccount` is not found?

Reply:
@alafanechere I actually think there is no exception anymore with this method, because this initialisation of `AdAccount` itself does not make any calls as far as I understand, and it only errors on subsequent calls to methods on this AdAccount. As such we can probably just remove the `except` block. Does that sound reasonable?